content/concepts/consensus-network/amendments/known-amendments.ja.md

```diff
@@ -473,6 +475,26 @@ Fixes a bug in unused code for estimating the ratio of input to output of indivi
 
 This amendment has no known impact on transaction processing.
 
+## fixRemoveNFTokenAutoTrustLine
+[fixRemoveNFTokenAutoTrustLine]: #fixremovenftokenautotrustline
+
+
+| Amendment ID                                                     | ステータス |
+|:-----------------------------------------------------------------|:---------|
+| DF8B4536989BDACE3F934F29423848B9F1D76D09BE6A1FCFE7E7F06AA26ABEAD | 投票中 |
+
+<!-- TODO: translate amendment description -->
+
+Removes the `tfTrustLine` setting on [non-fungible tokens](non-fungible-tokens.html), to protect against a denial of service attack on issuers using this flag. With this amendment enabled, an [NFTokenMint transaction](nftokenmint.html) with the `tfTrustLine` flag enabled is considered invalid and cannot be confirmed by consensus; therefore, `NFToken` objects cannot be minted with the flag.
```

Comment:
That's interesting. It's very unintuitive to me that it would be pronounced "Nif-Token" because I hear people talking about "enn eff tees" constantly. But OK, I'll make this change for consistency.